File: 백준/Bronze/33162. 散歩 （Walking）/散歩 （Walking）.cc
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;
    int cnt = 0;
    for (int i = 1; i <= n; i++)
    {
        if (i % 2 != 0)
        {
            cnt += 3;
        }
        else
        {
            cnt -= 2;
        }
    }
    cout << cnt << endl;
    return 0;
}

File: 백준/Bronze/33162. 散歩 （Walking）/散歩 （Walking）.rs
use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Write};

fn main() {
    let mut reader = BufReader::new(stdin().lock());
    let mut writer = BufWriter::new(stdout().lock());

    let mut input = reader.lines();

    if let Some(Ok(line)) = input.next() {
        let n = line.parse::<usize>().unwrap();
        let mut cnt = 0;
        //행동 a :3 진행
        //행동 b : 2m 돌아감
        for i in 1..=n {
            if i % 2 != 0 {
                cnt += 3;
            } else {
                cnt -= 2;
            }
        }

        writeln!(writer, "{cnt}").unwrap();
    }
    writer.flush().unwrap();
}
